applications/rag/workloads.tfvars

```diff
@@ -42,7 +42,7 @@ jupyter_service_account = "jupyter-system-account"
 dataset_embeddings_table_name = "googlemaps_reviews_db"
 
 ## IAP config
-add_auth                = false # Set to true when using auth with IAP
+add_auth                = var.add_auth # Set to true when using auth with IAP
```

Comment:
this should be true or false, not a var. this is workloads.tfvars where the user sets explicit values.

i would set it to false unless you verified it works E2E with RAG. we can set it to true in a followup PR.

Reply:
I have tested E2E, we can modify it to true after I push the image to repo, We have a tricky thing is that I can not make this code change and image updated the same time. So you can see I have a legacy support to ensure it will not break anything. But for sure IAP will be the default setting for jupyter. I will add it in next PR

Reply:
sorry, can we just explicitly set this to "false" instead?

Reply:
and update to true in the next PR

Reply:
changed to false